src/index.php

```diff
@@ -26,6 +26,12 @@
 // get user from url query string
 $user = $_REQUEST["user"];
 
+// redirect to sample site
+if(!isset($_REQUEST["user"]){
```

Comment:
Missing ')' and should have spaces around the expression:

```suggestion
if (!isset($_REQUEST["user"])) {
```